AddNewRecipes/Program.cs

```diff
@@ -89,27 +90,27 @@ public static void RunPatch(IPatcherState<ISkyrimMod, ISkyrimModGetter> state)
                     {
                         prefix = "Poison";
                         type = 1;
-                        if (mgeflist.Count() <= poisonSkipThreshold)
```

Comment:
This is a very simple change, but a very important change to understand.
What is a list?  It's basically an array that can grow.
A list (and any other container, for that matter) knows how many items it has, and you can just ask it and it'll tell you near instantly.  `list.Count`

Linq has a call you can make, which looks very similar.  `list.Count()`.  What's the difference?

The Linq version is nice, because it will work for ANY enumerable.  It'll work for any thing you can loop over, even if it doesn't know how many things it has.  How does it do this?  Well, it literally loops over all the things, and counts on its hands.   That takes work, but it's able to do it on anything.

So if you compare the line you wrote to the line I changed it to:
- My line returns near instant, because it's asking List its Count, which is a near instant return.  It knows, and so it just tells me.
- Your line of Count() instead uses Linq, which tells it to loop over every item in the list, counting the number of items on its hand.   Imagine if a list had `int.MaxValue` items.  You're literally looping over 6 billion things and counting one by one.  That's a lot of work, just to get a List's count. 

The fundamental takeaway is that Linq is powerful.  But it fundamentally has to loop over everything to do its job.  There's often many other ways to do things more optimally, with this being a prime example.